client/src/Calendar.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+
+let Calendar = (props) => (
+  <div>
+  <table>
+    <tr>
+      <th colSpan="7">September 2018</th>
+    </tr>
+    <tr>
+      <td>Su</td>
+      <td>Mo</td>
+      <td>Tu</td>
```

Comment:
I think if your calendar as an array of arrays, there's a better way to render that dynamically without hardcoding all the dates.